cmd/registry/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	_ "github.com/docker/distribution/registry/storage/driver/filesystem"
 	_ "github.com/docker/distribution/registry/storage/driver/inmemory"
 	_ "github.com/docker/distribution/registry/storage/driver/middleware/cloudfront"
+	_ "github.com/docker/distribution/registry/storage/driver/rados"
```

Comment:
Just have a file in the main package, `rados.go` with following in it:

```go
// +build include_rados

package main

import _ "github.com/docker/distribution/registry/storage/driver/rados"
```

I'd rather not have an empty package laying around, if at all possible.